temba/contacts/tests.py

```diff
@@ -347,7 +347,7 @@ def test_evaluate_dynamic_groups_from_flow(self):
                 ContactField.get_or_create(self.org, self.admin, key, value_type=Value.TYPE_NUMBER)
                 ContactGroup.create_dynamic(self.org, self.admin, "Group %s" % (key), "(%s > 10)" % key)
 
-        with QueryTracker(assert_query_count=110, stack_count=16, skip_unique_queries=False):
+        with QueryTracker(assert_query_count=118, stack_count=16, skip_unique_queries=False):
```

Comment:
this is a funny one ... it's related to this change https://github.com/rapidpro/rapidpro/pull/792/files#diff-8a47789b2c9de7beb2c22be6a4d78f2eR670 ... updated CampaignEvent query .. 

in the original form it was never evaluated and it did not execute a query, maybe it was cached internally, but I could not figure out where

extra 8 queries are all similar to ```SELECT "campaigns_campaignevent"."id", "campaigns_campaignevent"."is_active", "campaigns_campaignevent"."created_by_id", "campaigns_campaignevent"."created_on", "campaigns_campaignevent"."modified_by_id", "campaigns_campaignevent"."modified_on", "campaigns_campaignevent"."uuid", "campaigns_campaignevent"."campaign_id", "campaigns_campaignevent"."offset", "campaigns_campaignevent"."unit", "campaigns_campaignevent"."relative_to_id", "campaigns_campaignevent"."flow_id", "campaigns_campaignevent"."event_type", "campaigns_campaignevent"."message", "campaigns_campaignevent"."delivery_hour" FROM "campaigns_campaignevent" INNER JOIN "campaigns_campaign" ON ("campaigns_campaignevent"."campaign_id" = "campaigns_campaign"."id") INNER JOIN "contacts_contactfield" ON ("campaigns_campaignevent"."relative_to_id" = "contacts_contactfield"."id") WHERE ("campaigns_campaign"."group_id" IN (SELECT U0."id" FROM "contacts_contactgroup" U0 INNER JOIN "contacts_contactgroup_contacts" U1 ON (U0."id" = U1."contactgroup_id") WHERE (U1."contact_id" = 36 AND U0."group_type" = 'U')) AND "campaigns_campaign"."is_archived" = false AND "campaigns_campaignevent"."is_active" = true AND "contacts_contactfield"."key" IN ('total_calls_made'))```